eagle-topology-check/eagle-topology-app/src/main/java/org/apache/eagle/topology/TopologyCheckAppConfig.java

```diff
@@ -32,6 +32,8 @@
 
     public static final String TOPOLOGY_DATA_FETCH_SPOUT_NAME = "topologyDataFetcherSpout";
     public static final String TOPOLOGY_ENTITY_PERSIST_BOLT_NAME = "topologyEntityPersistBolt";
+    public static final String PARSE_BOLT_NAME = "parserBolt";
+    public static final String KAFKA_SINK_BOLT_NAME = "kafkaSinkBolt";
```

Comment:
Please use `sinkBolt` directly instead of `kafkaSinkBolt` as `kafka` is the underly implementation of `StreamSink` term.